atm.js

```diff
@@ -1,3 +1,191 @@
+/*********
+Pseudo Code At Bottom
+**********/
+
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+/*******
+CHECKINGS
+********/
+
+  var checkingTotal = 0;
+  if (checkingTotal === 0) {
+    $("#checking .balance").css("background", "rgb(232, 63, 63)")
+  }
+
+  $("#checking .deposit").on("click", function (evt) {
+    evt.preventDefault();
+    var depositAmount = $("#checking .input").val()
+    if (isNaN(depositAmount)) {
+      $("#checking .balance").html("<p class='smallText'>Please enter an amount</p>");
+      $("#checking .balance").css("background", "rgb(232, 63, 63)");
+    } else {
+      $("#checking .balance").css("background", "#E3E3E3");
+      depositFunction(depositAmount);
+      $(".input").val("");
+    }
+  });
+
+  $("#checking .withdraw").on("click", function (evt) {
+    evt.preventDefault();
+    var withdrawAmount = $("#checking .input").val()
+    if (withdrawAmount == "") {
+      $("#checking .balance").html("<p class='smallText'>Please enter an amount</p>");
+      $("#checking .balance").css("background", "rgb(232, 63, 63)");
```

Comment:
I might have done some method chaining here to save up some lines of code. But I like that you're thinking about user experience here with the custom messaging.